scripts/resultsSum.pl

```diff
@@ -231,6 +231,8 @@ sub resultReporter {
 		my $platformParam = "";
 		if (exists $spec2platform->{$spec}) {
 			$platformParam = "&PLATFORM=" . $spec2platform->{$spec};
+			# spec fileuses aarch32, but Grinder needs arm as the parameter
+			$platformParam =~ 's/aarch32_/arm_/';
```

Comment:
If the aarch32 is replaced by arm, I believe we need to update the map file: https://github.com/adoptium/TKG/blob/master/resources/buildPlatformMap.properties#L24, instead of change it here. @llxia What do you think?

Reply:
I wasn't expecting that we change it universally for now, just set the "correct" Grinder parameter as a tactical fix to it giving an invalid link. If we choose to do a larger change later that should probably be a separate PR

Reply:
This is not the correct place to change it. The update should be in https://github.com/adoptium/TKG/blob/master/resources/buildPlatformMap.properties#L24 as @renfeiw mentioned above. It is one line change.

Reply:
This is not the correct place to change it. The update should be in https://github.com/adoptium/TKG/blob/master/resources/buildPlatformMap.properties#L24 as @renfeiw mentioned above. It is one line change.

Reply:
Will that not change other things? I am not looking to use this PR to change what the platform is called universally, only to fix a situation where an invalid parameter is appearing on re-run jobs. Will the `buildPlatformMap.properties` only affect that, or cause other (potentially breaking) changes if that was the only place it was changed?